translations/en-us.yml

```
@@ -917,7 +919,7 @@ institutions:
               reply_to_label: 'Allow reply to sender address'
               send_message_prompt:
                 title: "Can't Request Access"
-                message: "This user has their request access feature turned off. Reach out to the user via the User tab by pressing the 'Back' button below."
+                message: 'The user has their request access feature turned off. Reach out to the user by sending an email message by pressing the “Back button below.'
```

Does this need a closing quotation mark after `"Back`?

```suggestion
                message: 'The user has their request access feature turned off. Reach out to the user by sending an email message by pressing the “Back" button below.'
```